src/handlers/user.rs

```diff
@@ -82,11 +87,18 @@ pub async fn get_user_without_auth(
     Ok(Json(user))
 }
 
+const MAX_BIO_LENGTH: usize = 10;
```

Comment:
adjust this to a higher number, like 5000

Reply:
This is a small nitpick. But you can also move it inside the handler function. We use this in only one place compiler should optimize it into a static variable. even if it doesn't, it's such a small performance impact. 

The reason I want this is that I did this in other functions and it should make it consistent with the rest of the code base.